Chinese/老司机/老司机.md

```diff
@@ -0,0 +1 @@
+因为涉嫌无证驾驶被秋名山飙车大会拘留，警方在其后备箱中发现多个违法套牌：MIAD-530、SW-072、MIDD-944、LADY-077、SW-186、STAR444、T28-184、dvdes-635、BOD-277、ARMG-014、JUC-579、BBI-142、MILD-716、FSLV-002、CRS-S014、ODFW-006、SOE-837、Nhdta-141、NADE-783、MIDD-532、IPTD-748、IESP-144、crpd-222、GAR-280、BW248、MXGS173、MIAD-530、RCT-402。秋名山老司机表示，发现该套牌严重影响了秋名山飙车大赛
```

Comment:
Thanks for your advice!
This paragraph is just a piece of article I found on the web, and it actually has not much meaning. Just use this to practice.